wicket-core/src/main/java/org/apache/wicket/page/DefaultPageLockManager.java

```diff
@@ -133,9 +133,10 @@ public void lockPage(int pageId) throws CouldNotLockPageException
 		}
 		else
 		{
+			final String previousThreadName = previous != null ? previous.getThread().getName() : "N/A";
+			final Thread previousThread = previous != null ? previous.getThread() : null;
```

Comment:
nit: swap these two lines and simplify the latter to: `final String previousThreadName = previousThread != null ? previousThread.getName() : "N/A";`